gaia-list.js

```diff
@@ -29,11 +29,23 @@ module.exports = component.register('gaia-list', {
     this.setupShadowRoot();
     this.els = { inner: this.shadowRoot.querySelector('.inner') };
     this.addEventListener('click', this.onPointerDown);
-    this.makeAccessible();
+    // offload the initiate operation to save loadtime
+    setTimeout(() => {
+      this.makeAccessible();
```

Comment:
Can you give more detail in this comment. IE. *What* is expensive?

NIT: `setTimeout(() => this.makeAccessible());`